marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -96,16 +96,6 @@ class Flow {
     return docs;
   }
 
-  deleteFlow(flowName) {
```

Comment:
I deleted this because a) it wasn't being used anywhere, and b) it didn't work - "this.hubUtils" did not resolve.

Reply:
Should probably fix this or, comment it out/leave it and bug log it not remove it, as the new app is going to be atomicly changing things on the server instead of doing full reloads.

Reply:
I believe this is being handled now by artifacts/core.sjs